### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by omgms** [frereit/TensorflowHandwritingRecognition#9]
Hi just want to check how are you matching the labels to the training data? 

Like for e.g. inside class 4e, it contains letter 'N', so why is the label 4e instead of N?

**Comment by chouxscream** [comment]
ASCII code, in hexadecimal
https://en.wikipedia.org/wiki/ASCII

**Comment by lavanyaakella** [comment]
Hi 
Im getting an error in line 184, i.e epochs = int(sys.argv[1])
the error is : list index out of range
What should be done?
